skbio/stats/ordination/__init__.py

```diff
@@ -126,11 +127,11 @@
 from ._redundancy_analysis import rda
 from ._correspondence_analysis import ca
 from ._canonical_correspondence_analysis import cca
-from ._principal_coordinate_analysis import pcoa
+from ._principal_coordinate_analysis import pcoa, pcoa_biplot
 from ._ordination_results import OrdinationResults
 from ._utils import (mean_and_std, scale, svd_rank, corr, e_matrix, f_matrix)
 
-__all__ = ['ca', 'rda', 'cca', 'pcoa', 'OrdinationResults',
+__all__ = ['ca', 'rda', 'cca', 'pcoa', 'pcoa_biplot', 'OrdinationResults',
```

Comment:
Might be something to bring up to the rest of the group -- would it make more sense to pass in one optional argument into `pcoa` to create biplots vs adding a new method?  Maybe we could have something like
```python
def pcoa(distance_matrix, y=None):
'''
If y is specified, biplots will be generated ...
'''
```

Reply:
The interface mimics what `ape` does. Which is likely for cases where a *large* PCoA has already been computed. In that case, there's no reason for recomputing the eigendecomposition.

Reply:
@antgonza do you have any thoughts on this?  I was thinking that it maybe a good idea to incorporate this functionality directly into the pcoa function.

Reply:
I think going what's implemented is the easiest. Note that I agree that having the pcoa calculation and the pcoa_biplot in different steps makes sense due to large distance matrices. BTW that's how biplots used to work in Qiime1 too.